ocp_resources/node_network_configuration_policy.py

```diff
@@ -307,8 +324,11 @@ def wait_for_interface_deleted(self):
     def validate_create(self):
         for pod in self.worker_pods:
             for bridge in self.ifaces:
-                node_network_state = NodeNetworkState(name=pod.node.name)
-                node_network_state.wait_until_up(name=bridge["name"])
+                if (
+                    "capture" not in bridge["name"]
+                ):  # Wrapper can't tell a capture iface's actual name
```

Comment:
don't we set the name?
capture={'first-nic': 'interfaces.name=="ens8"',
              'second-nic': 'interfaces.name=="ens9"'},

Reply:
That was just a simple example. Usually, like when taking over a bridge's old port, you won't know the port name.
I.e, capture={..., 'interface': 'capture.br1.interfaces.0.bridge.port.0.name'}